src/RSP_LoadMatrix.cpp

```diff
@@ -0,0 +1,37 @@
+#include <algorithm>
+#include <cstring>
+#include "Debug.h"
+#include "RSP.h"
+#include "RDP.h"
+#include "N64.h"
+#include "F3D.h"
+#include "Turbo3D.h"
+#include "VI.h"
+#include "Combiner.h"
+#include "FrameBuffer.h"
+#include "DepthBuffer.h"
+#include "FrameBufferInfo.h"
+#include "GBI.h"
+#include "PluginAPI.h"
+#include "Config.h"
+#include "TextureFilterHandler.h"
+#include "DisplayWindow.h"
+
```

Comment:
Android Studio will grey out unnecessary includes. It would help with things such is this.